src/components/Home.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import styles from './Home.module.css';
+
+const Home = () => (
+  <div className={styles.home}>
+    <h2>Welcome to our page!</h2>
+    <p>
+      Lorem ipsum dolor sit amet consectetur adipisicing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Ipsu
+      <br />
+      <br />
+      Lorem ipsum dolor sit amet consectetur adipisicing elit, sed do eiusmod tempor incididunt
+      ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco
+      laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in
+      voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat
+      non proident, sunt in culpa qui officia deserunt mollit anim id est laborum. Ipsu
+    </p>
+  </div>
+);
```

Comment:
- [ ] Please make sure to use React basic styling instead of CSS modules as requested in the project requirements.

![image](https://github.com/mailsg/Math-Magicians/assets/83803414/699ddc4b-412a-4b4e-b4b0-7b1cad6b1ea2)

Reply:
Used React basic styling by importing CSS.